js/webapp.js

```diff
@@ -433,6 +429,45 @@ ImportManager.prototype = {
   
 }
 
+function getLoopbackDefaults(webappOptions) {
+  //if this isn't here, throw
+  let nodeConfig = webappOptions.serverConfig.node;
+
+  let getPreferredAddress = function(addresses) {
```

Comment:
This looks like it needs to be moved to util.js

1. It doesn't seem to refer to the local variables of the surrounding function 
2. It can be usable elsewhere
3. In this PR: https://github.com/zowe/zss-auth/pull/8/files#diff-9c4028ca3fee8475c7b0f0a4f5a92580R115 I actually used the 'ipaddr.js' library to check if an IP address is loopback, because e.g. 127.0.0.2 is also a valid loopback addr.

Reply:
OK - wasn't sure where to put it so that makes sense.